front-end/src/app/shared/components/transaction-type-base/transaction-form.utils.ts

```diff
@@ -89,7 +89,7 @@ export class TransactionFormUtils {
         });
     }
 
-    if (transactionType.showAggregate) {
+    if (transactionType.showAggregate && !transaction.force_unaggregated) {
```

Comment:
looks like the aggregate still needs to show for these transactions.  the difference being that we don't add the amount of the unaggegated transaction to the aggregate.  seems like we could update `updateAggregate` to add $0 if the transaction is unaggregated

Reply:
I believe that is how it is working currently. The !transaction.force_unaggregated above simply keeps values and changes made to the Amount in the form from adding to the displayed Aggregate, which is the value from the previous transaction.